tests/pii_replacer/test_edit.py

```
@@ -311,3 +312,20 @@ def test_forced(self):
+    def test_reset_timing_excludes_preprocessing_from_row_speed(self):
+        with patch(
+            "nemo_safe_synthesizer.pii_replacer.data_editor.edit.monotonic",
+            side_effect=[10.0, 10.0, 310.0, 312.0, 313.0],
+        ):
+            progress = ProgressLog(30)
+            progress.status.row_n = 20
+            progress.reset_timing()
+            with self.assertLogs() as logs:
+                progress.log_throttled(force=True)
+
+        ctx = getattr(logs.records[0], "ctx")
```

Ruff (0.15.21)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)